pkg/git/operations.go

```diff
@@ -35,6 +35,8 @@ var allowedEnvVars = []string{
 	// these are for Google Cloud SDK to find its files (which will
 	// have to be mounted, if running in a container)
 	"CLOUDSDK_CONFIG", "CLOUDSDK_PYTHON",
+	// these are used for supporting HTTPS basic auth
+	"GIT_AUTHUSER", "GIT_AUTHKEY",
```

Comment:
Are you certain these need to be exposed to `git` -- won't it just be given the URL with those values already interpolated?

Reply:
You are right, this is a left-over from the previous approach I attempted.